### PR TITLE
sdm845: SOCSmbiosInfoLib: Use pcd to update soc description

### PR DESCRIPTION
**Pull request by qaz6750**
* Signed-off-by: qaz6750 <2838632319@qq.com>

#### SDM845
* Use pcd to update soc description 
### Checklist

* [Y] Have you tested the change you are submitting?
* [Y] Is the commits history nice and clean?

### CODE REVIEW

**Comment by gus33000** [comment]
Official BSP of the time for 850 didn't have the frequency appended to the cpu name (this came later with 8180/7180 instead)

only Snapdragon (TM) 850

**Comment by gus33000** [comment]
image
Snapdragon 8c as well, not just 8cx/7c

**Comment by gus33000** [comment]
image
sorry looks like it was backported eventually, discard what i said

File: Silicon/Qualcomm/sdm845/Library/SOCSmbiosInfoLib/SOCSmbiosInfo.c [hunk 1]
```diff
@@ -120,7 +120,7 @@ SMBIOS_TABLE_TYPE4 mProcessorInfoType4_a55 = {
 CHAR8 mCpuName[128] = "Qualcomm Snapdragon 845";
 
 CHAR8 *mProcessorInfoType4Strings[] = {
-    "BGA", "Qualcomm", "Snapdragon 845", NULL};
+    "BGA", "Qualcomm", "Not Specified", NULL};
```
Comment (MollySophia) [inline Silicon/Qualcomm/sdm845/Library/SOCSmbiosInfoLib/SOCSmbiosInfo.c:123]: This change seems to have no actual use, but it's okay :P

**Review by MollySophia — APPROVED** [review]

**Pull request merged**